tests/light/functional_tests/destination_drivers/network_destination/test_tls_verifier_reload_crash.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2024 Axoflow
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+from src.common.file import copy_shared_file
+
+
+def test_tls_verifier_reload_crash(config, syslog_ng, port_allocator, testcase_parameters):
+    ca = copy_shared_file(testcase_parameters, "valid-ca.crt")
+
+    network_source = config.create_network_source(port=port_allocator())
+    network_destination = config.create_network_destination(
```

Comment:
Maybe in the variable name we should highlight that this is a `tls` destination

Reply:
Added `tls` to the variable name.